src/librustc_codegen_ssa/Cargo.toml

```diff
@@ -31,4 +31,5 @@ rustc_errors = { path = "../librustc_errors" }
 rustc_fs_util = { path = "../librustc_fs_util" }
 rustc_incremental = { path = "../librustc_incremental" }
 rustc_index = { path = "../librustc_index" }
+rustc_mir = { path = "../librustc_mir" }
```

Comment:
Note that if you make `const_caller_location` a query then this disappears (this is a dependency we really don't want to add because it serializes two relatively large crates).